<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Command Center</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Resources/nSOTScannerResources.qrc">
    <normaloff>:/nSOTScanner/Pictures/jj_schema.svg</normaloff>:/nSOTScanner/Pictures/jj_schema.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget
{
background-color: rgb(0, 0, 0);
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,2,2,1,1" columnstretch="2,5,2,3,2,5,2">
    <item row="5" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="3">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>28</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pushButton_TransferLeft">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#pushButton_TransferLeft{
image:url(:/nSOTScanner/Pictures/ArrowLeft.png);
background: black;
}

QPushButton:pressed#pushButton_TransferLeft{
image:url(:/nSOTScanner/Pictures/ArrowLeft.png);
background: black;
border: 2px rgb(131,131,131);
border-radius: 5px
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="pushButton_TransferRight">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#pushButton_TransferRight{
image:url(:/nSOTScanner/Pictures/ArrowRight.png);
background: black;
}

QPushButton:pressed#pushButton_TransferRight{
image:url(:/nSOTScanner/Pictures/ArrowRight.png);
background: black;
border: 2px rgb(131,131,131);
border-radius: 5px
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="6">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="pushButton_Process">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:pressed#pushButton_Process{
color: rgb(0,0,0);
background-color:rgb(131,131,131);
border: 1px solid rgb(0,0,0);
border-radius: 5px
}

QPushButton#pushButton_Process{
color: rgb(131,131,131);
background-color:rgb(0,0,0);
border: 2px solid  rgb(131,131,131);
border-radius: 5px
}</string>
      </property>
      <property name="text">
       <string>Process</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QPushButton" name="pushButton_Cancel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:pressed#pushButton_Cancel{
color: rgb(0,0,0);
background-color:rgb(131,131,131);
border: 1px solid rgb(0,0,0);
border-radius: 5px
}

QPushButton#pushButton_Cancel{
color: rgb(131,131,131);
background-color:rgb(0,0,0);
border: 2px solid  rgb(131,131,131);
border-radius: 5px
}</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>28</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1" colspan="5">
     <widget class="QLabel" name="label_Operation">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:grey</string>
      </property>
      <property name="text">
       <string>Operation</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" rowspan="4" colspan="3">
     <widget class="QListWidget" name="listWidget_PlotsListA">
      <property name="styleSheet">
       <string notr="true">QListWidget#listWidget_PlotsListA{
color: rgb(131,131,131);
background-color:black;
border: 2px solid rgb(200,50,50);
border-radius: 5px
}

QScrollBar:horizontal {
   border: 0px;
   background: black;
   height: 15px;
}
QScrollBar:vertical {
   border: 0px;
   background: black;
   width: 15px;
}</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4" rowspan="4" colspan="3">
     <widget class="QListWidget" name="listWidget_PlotsListB">
      <property name="styleSheet">
       <string notr="true">QListWidget#listWidget_PlotsListB{
color: rgb(131,131,131);
background-color:rgb(0,0,0);
border: 2px solid rgb(50,50,200);
border-radius: 5px
}

QScrollBar:horizontal {
   border: 0px;
   background: black;
   height: 15px;
}
QScrollBar:vertical {
   border: 0px;
   background: black;
   width: 15px;
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../Resources/nSOTScannerResources.qrc"/>
 </resources>
 <connections/>
</ui>
